content/pages/feed/feed.md

```diff
@@ -0,0 +1,40 @@
+---
+permalink: /feed.xml
+skip_index: true
+---
+{% assign selected_posts = collections.posts | reverse | slice:0, 10 -%}
+<feed xmlns="http://www.w3.org/2005/Atom">
+<link href="{{ site.host }}/feed.xml" rel="self" type="application/atom+xml"/>
```

Comment:
change: In order to stay consistent with the rest of the site's links, _I believe_ anything that uses {{ site.host }} should be changed to be using the `url` filter, like:

```liquid
"{{ "/feed.xml" | url }}"
```

Reply:
I tried using that shortcode but it adds classes to the output that don't belong in the XML feed.

Reply:
@jasnakai What do you mean by "classes"?

Reply:
This is what gets added when using taha shortcode: class="usa-link usa-link--external"

Reply:
Also, the generated url from the shortcode doesn't contain the full path: href="/feed.xml"

Reply:
That is...not expected behavior. Ok, we'll approve and note that issue for later. Thanks!